var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -126,6 +96,11 @@ def configure(self, spec, prefix):
 
         # Variants that affect every precision
         if '+openmp' in spec:
+            # Note: Apple's Clang does not support OpenMP.
+            if spec.satisfies('%clang'):
+                ver = str(self.compiler.version)
+                if ver.endswith('-apple'):
+                    raise InstallError("Apple's clang does not support OpenMP")
```

Comment:
This can be converted to a conflict for `%apple-clang`, the compiler is no longer called `%clang`

Reply:
Currently apple-clang section is redundant, already it is handled in the conflict checks.
So removed that section under base fftw recipe.